src/utils/computeSurroundingTicks.ts

```diff
@@ -10,7 +10,7 @@ export default function computeSurroundingTicks(
   token0: Token,
   token1: Token,
   activeTickProcessed: TickProcessed,
-  sortedTickData: TickData[],
+  sortedTickData: readonly TickData[],
```

Comment:
what's the purpose of this change?

Reply:
relay's generated data type AllV3TicksQuery$data returns a readonly. I deleted this type and refactored to use the generated types instead